engine/src/main/java/org/terasology/rendering/dag/nodes/OutlineNode.java

```diff
@@ -76,9 +125,47 @@ public OutlineNode(Context context) {
     public void process() {
         PerformanceMonitor.startActivity("rendering/outline");
 
+        // Common Shader Parameters
+
+        outlineMaterial.setFloat("viewingDistance", renderingConfig.getViewDistance().getChunkDistance().x * 8.0f, true);
+
+        outlineMaterial.setFloat("daylight", backdropProvider.getDaylight(), true);
+        outlineMaterial.setFloat("tick", worldRenderer.getMillisecondsSinceRenderingStart(), true);
+        outlineMaterial.setFloat("sunlightValueAtPlayerPos", worldRenderer.getTimeSmoothedMainLightIntensity(), true);
+
+        cameraDir = activeCamera.getViewingDirection();
+        cameraPosition = activeCamera.getPosition();
+
+        outlineMaterial.setFloat("swimming", activeCamera.isUnderWater() ? 1.0f : 0.0f, true);
+        outlineMaterial.setFloat3("cameraPosition", cameraPosition.x, cameraPosition.y, cameraPosition.z, true);
+        outlineMaterial.setFloat3("cameraDirection", cameraDir.x, cameraDir.y, cameraDir.z, true);
+        outlineMaterial.setFloat3("cameraParameters", activeCamera.getzNear(), activeCamera.getzFar(), 0.0f, true);
+
+        sunDirection = backdropProvider.getSunDirection(false);
+        outlineMaterial.setFloat3("sunVec", sunDirection.x, sunDirection.y, sunDirection.z, true);
+
+        outlineMaterial.setFloat("time", worldProvider.getTime().getDays(), true);
+
+        // Specific Shader Parameters
+
+        outlineMaterial.setFloat("texelWidth", 1.0f / sceneOpaqueFboWidth);
+        outlineMaterial.setFloat("texelHeight", 1.0f / sceneOpaqueFboHeight);
+
+        outlineMaterial.setFloat("pixelOffsetX", pixelOffsetX);
+        outlineMaterial.setFloat("pixelOffsetY", pixelOffsetY);
+
+        // Actual Node Processing
+
         renderFullscreenQuad();
 
         PerformanceMonitor.endActivity();
 
     }
+
+    @Override
+    public void update() {
```

Comment:
Yeah, it would seem so. This is, as you say, very interesting, because although I made the resizing fixes on low settings because of my PC hardware, I'm fairly certain I tested it on high settings as well (as did Cerv).